python/ql/test/2/query-tests/Expressions/expressions_test.py

```diff
@@ -1,7 +1,18 @@
-
 def use_of_apply(func, args):
     apply(func, args)
 
+
 def use_of_input():
-    return input()
+    return input() # NOT OK
+
+
+def not_use_of_input():
+    input = raw_input
+    return input() # OK
+
 
+if __name__ == "__main__":
+    # if you enter 4+4 each time, you'll see that results are: 8, '4+4', 8
+    print("result:", use_of_input())
+    print("result:", not_use_of_input())
+    print("result:", use_of_input())
```

Comment:
At first, I was like "why test this again?", but I'm guessing now that this is to test that `input` is unaffected by the call to `not_use_of_input`.